dali/pipeline/operator/op_schema.h

```diff
@@ -456,6 +456,15 @@ class DLL_PUBLIC OpSchema {
     return *this;
   }
 
+  /**
+   * @brief Informs that the data passes though this operator unchanged, only
+   *        the metadata is affected.
+   */
```

Comment:
Please add some info about what should be inside the `inout`.
How this going to work with operators that may have a different amount of in/outs depending on arguments.